hack/devtools/rp_dev_helper.sh

```diff
@@ -1,39 +1,471 @@
+    if ! check_acr_repo ${dst_acr_name} ${repository} "true" ${tag} ;then
+        az acr import --name ${dst_acr_name}.azurecr.io/${repository}:${tag} --source linuxgeneva-microsoft.azurecr.io/${repository}:${tag}
+        log "🟢✈️ Imported repository '${repository}' to ACR '${dst_acr_name}'"
+    else
+        log "⏭️📦 Skip importing repository '${repository}' to ACR '${dst_acr_name}', since it already exist with tag ${tag}. Import can not run twice"
+    fi
+}
+
+check_keyvault_certificate() {
+    err_str="Usage $0 <KEYVAULT> <CERTIFICATE> [SKIP_DEPLOYMENTS]. Please try again"
+    local key_vault=${1?$err_str}
+    local certificate=${2?$err_str}
+    local skip_deployments=${3:-"true"}
+
+    # Don't skip deployment creation when skip_deployments was set to 'false'
+    if is_it_boolean $skip_deployments && ! $skip_deployments; then
+        abort "'skip_deployments' was set to 'false'. ❌⏩ Don't skip keyvault's certificate $certificate import"
+    fi
+    
+    # Check if the Key Vault exists
+    if ! az keyvault show --name "$key_vault" >/dev/null 2>&1; then
+        abort "🔴❌💼 Error: Key Vault '$key_vault' does not exist."
+    fi
+
+    certificate_info="$(az keyvault certificate show --vault-name "$key_vault" --name "$certificate" 2>/dev/null)"
+    if [ -z "${certificate_info}" ]; then
+        log "🔴❌💼 Certificate '$certificate' in Key Vault '$key_vault' does not exist."
+        return 1
+    fi
+
+    # check_jq_installed - Might not needed
+
+    local -r attributes_enabled="$( jq -r '.attributes.enabled' <<< $certificate_info)"
+    local -r attributes_expires="$( jq -r '.attributes.expires' <<< $certificate_info)"
+    # we don't validate the thumbprint for exact match
+    if $attributes_enabled; then 
+        log "🟢💼 Certificate '$certificate' in Key Vault '$key_vault' exists and is enabled with expiration date '$attributes_expires'."
+    else
+        abort "🔴💼 Certificate '$certificate' in Key Vault '$key_vault' exists but is not enabled."
+    fi
 }
 
-# Example usage
-# get_digest_tag "FluentbitImage"
-# copy_digest_tag "<PULL_SECRET>" "src_acr_name" "dst_acr_name" "$(get_digest_tag FluentbitImage)"
+skip_and_import_certificates(){
+    err_str="Usage $0 <KEYVAULT>  <CERTIFICATEs...> <SECRET_FILES...> [SKIP_DEPLOYMENTS]. Please try again"
+    local keyVault=${1?$err_str}
+
+    # Extract the certificates and secret_files arraies based on half the remaining arguments
+    local num_certs=$((($# -1) / 2))
+    local -a certificates=("${@:2:$num_certs}")
+    local -a secret_files=("${@:$((num_certs + 2)):$num_certs}")
+    # Optional flag (last argument)
+    local skip_deployments="${@: -1}"
+
+    # Don't skip deployment creation when skip_deployments was set to "false" 
+    if is_it_boolean $skip_deployments && ! $skip_deployments; then
+        abort "'skip_deployments' was set to 'false'. ❌⏩ Don't skip certs import to keyVault $keyVault"
+    fi
+
+    for i in "${!certificates[@]}"; do
```

Comment:
Expand array value as a string directly.

```suggestion
    for i in ${!certificates[*]}; do
```